packages/mdc-ripple/README.md

```diff
@@ -52,30 +64,32 @@ npm install --save @material/ripple
 
 ## Usage
 
-### Adding Ripple styles
+A ripple can be applied to a variety of elements to represent interactive surfaces. Several MDC Web components, such as Button, FAB, Checkbox and Radio, also use ripples.
 
-General notes:
+A ripple can be added to an element through either a JavaScript or CSS-only implementation. When a ripple is initialized on an element using JS, it dynamically adds a `mdc-ripple-upgraded` class to that element. If ripple JS is not initialized but Sass mixins are included on the surface, the ripple uses a simpler CSS-only implementation which relies on `:hover`, `:focus`, and `:active`.
 
-* Ripple mixins can be applied to a variety of elements representing interactive surfaces. These mixins are also used by other MDC Web components such as Button, FAB, Checkbox, Radio, etc.
-* Surfaces for bounded ripples should have `overflow` set to `hidden`, while surfaces for unbounded ripples should have it set to `visible`
-* When a ripple is successfully initialized on an element using JS, it dynamically adds a `mdc-ripple-upgraded` class to that element. If ripple JS is not initialized but Sass mixins are included on the surface, the ripple will still work, but it uses a simpler, CSS-only implementation which relies on `:hover`, `:focus`, and `:active`.
+### CSS Classes
 
-#### Sass API
+CSS Class | Description
+--- | ---
+`mdc-ripple-surface` | Adds a ripple to the element
+`mdc-ripple-surface--primary` | Sets the ripple color to the theme primary color
+`mdc-ripple-surface--accent` | Sets the ripple color to the theme secondary color
 
-In order to fully style states as well as the ripple effect for pressed state, both `mdc-ripple` mixins below must be included, as well as either the basic or advanced `mdc-states` mixins documented below.
+### Sass Mixins
 
-Once these styles are in place for a component, it is feasible to further override only the parts necessary (e.g. `mdc-states` specifically) for specific variants (e.g. for flat vs. raised buttons).
+In order to fully style states as well as the ripple effect for pressed state, both `mdc-ripple` mixins, as well as either the basic or advanced `mdc-states` mixins must be included.
```

Comment:
maybe add a code example of using `mdc-ripple` and `mdc-states` in some CSS. This sentence is not very clear otherwise.